src/System.Text.Json/tests/Serialization/DictionaryTests.cs

```diff
@@ -65,9 +65,39 @@ public static void DuplicateKeysFail()
         }
 
         [Fact]
-        public static void DictionaryOfObjectFail()
```

Comment:
Can we keep this test please?

Reply:
We did, the test was looking to see that we failed gracefully with `<string, object>`, but mistakenly was missing a closing curly brace and wasn't testing the right path. The test now shows that the scenario works (which it didn't before).

Reply:
> mistakenly was missing a closing curly brace and wasn't testing the right path

I see, this was unintentionally incorrect. I thought the missing brace was intentional.